lib/error_report_notify.c

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: GPL-2.0
+
+/*
+ * Userspace notification interface for debugging tools.
+ *
+ * Provide two sysfs files:
```

Comment:
Isn't this a copy-paste from Kconfig help text?

Hmm, maybe the text here should be the more elaborate one, and the Kconfig help text be a summary. At the very least, I would probably remove the 'poll(POLLPRI)' from the Kconfig help text.

Or should it be the other way around?

I'd probably shorten this text here, as it doesn't show up in Documentation nor Kconfig help.

Reply:
> I would probably remove the 'poll(POLLPRI)' from the Kconfig help text.
There is no place in Documentation for this line, and I think people need to find this out somehow, so maybe keeping it in Kconfig will help?

Reply:
Keeping it in Kconfig makes sense, as this file is mainly for kernel devs. So, the redundancy could be resolved by removing it  here and leaving the Kconfig as-is?

I leave it to you. If you think both files need to mention this, I don't mind, but worry that if one of these is updated then the other will be out-of-date, or we have to keep changing 2 places to keep things in sync.